phonebook-api/src/main/java/com/telran/phonebookapi/controller/UserController.java

```diff
@@ -0,0 +1,29 @@
+package com.telran.phonebookapi.controller;
+
+import com.telran.phonebookapi.dto.UserDto;
+import com.telran.phonebookapi.entity.User;
+import com.telran.phonebookapi.service.ConfirmationTokenService;
+import com.telran.phonebookapi.service.UserService;
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+@RestController
+@AllArgsConstructor
+public class UserController {
+
+    private final UserService userService;
+    private final ConfirmationTokenService confirmationTokenService;
+
+    @PostMapping("/api/v1/registration")
+    public void create(@RequestBody @Valid UserDto userDto) {
+        userService.create(new User(userDto.email, userDto.password));
```

Comment:
the userService.create method should not accept ready user instance. Just pass in the email and password instead